src/admin-views/editor/recurring-warning.php

```diff
@@ -0,0 +1,20 @@
+<?php
+/**
+ * The template for the Series Pass event notice.
+ *
+ * @since TBD
+ *
+ * @var array $messages The warning message.
+ */
+
+?>
+<div class="ticket-editor-notice info info--background tec_ticket-panel__recurring-unsupported-warning" style="display: none">
+	<span class="dashicons dashicons-lightbulb"></span>
+	<div class="ticket-editor-notice_warning--messages">
+	<?php foreach ( $messages as $key => $message ) : ?>
+		<p class="ticket-editor-notice_warning--message <?php echo esc_attr( $key ); ?>">
+		<?php echo wp_kses_post( $message ); ?>
```

Comment:
Is there any reason to use `wp_kses_post` over a more targeted and faster-running `wp_kses` as it was before?